tests.bash

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash -x
+set -x #echo on
+set -e #exit at the first error
+
+MYPWD=$(pwd)
+
+killandsleep () {
+  kill -9 $(lsof -t -i:$1) || true
+  sleep 10
+}
+
+if [ $CIRCLE_NODE_INDEX -eq 0 ]
```

Comment:
Is it specified somewhere that we have 4 parallel runners? If not, may be good to enforce it so that it does not change under our feet and breaks tests (if there's a way)